Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM --platform=linux/amd64 clux/muslrust:stable as builder
+ENV CARGO_NET_GIT_FETCH_WITH_CLI=true
+WORKDIR /app
+COPY . .
+RUN rustup target add x86_64-unknown-linux-musl
+RUN cargo build --release --target x86_64-unknown-linux-musl
+RUN strip /app/target/x86_64-unknown-linux-musl/release/rindexer_cli
+
+FROM --platform=linux/amd64 node:lts as node-builder
+WORKDIR /app
+COPY . .
```

Comment:
I think copying the files/dirs needed specifically will prevent unwanted files from being copied over

alternatively you can ignore everything in `.dockerignore` except for the files you want to copy over. Example

```dockerignorr
# ignore everything
*

# except the following
!scripts/
!Dockerfile
!.dockerignore
!rindexer
```

https://github.com/o-az/rindexer-railway/blob/main/.dockerignore

Reply:
true but in the end, we'll copy over only the two static binaries into the final alpine image and just publish that to the registry.